### DISCUSSION TITLE
Welcome to microsoft-teams-apps-company-communicator Discussions!

### DISCUSSION THREAD

**Discussion by Mikhail2k15** [mtcmoscow/microsoft-teams-apps-company-communicator#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Alexandrovdi** [comment]
When i try add api on Expose an API in Application ID URI, sometimes it's go to error: "Failed to update application property. Error detail: Another object with the same value for property identifierUris already exists." How to fix it with out reinstalling the app.

**Reply by Mikhail2k15** [reply]
Какой вариант деплоймента используется? скриптом https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/Deployment-guide-powershell ? или обычный по инструкции?

Если речь про этот id, то где понадобилось его менять? 
![image](https://user-images.githubusercontent.com/11201670/108531399-b3885400-72e7-11eb-96ca-0551099bae7f.png)

**Reply by Alexandrovdi** [reply]
Развертывание происходит по инструкции. 
Да верно этот ИД я имел в виду.
В первый раз когда я пытаюсь его указать выходит выше указанная ошибка

**Reply by Mikhail2k15** [reply]
Так его название совпадает с URL azure front door сервисом, который в ресурной группе, то проверь он у тебя в ресурсной группе точно с таким урлом, потом проверь а нет ли у тебя еще одного приложения в AAD зарегистрировано где в техже настройках прописано тот же App ID URL
